android-core/build.gradle

```diff
@@ -7,7 +7,7 @@ android {
     compileSdk 34
```

Comment:
I'm not sure if the compileSdk version matters. Also, should we change Sdk in Mysignal as well? As it goes with BundleClient on a same device

Reply:
we will probably delete mysignal soon, so i'm leaving it alone. we want to stick with the 34 compileSdk. it does seem weird that min can be less, but it allows for doing exactly what we are doing: targeting a more modern SDK but still enabling older devices to still mostly work.

Reply:
wow I got it now